components/setting_item_max.jsx

```diff
@@ -165,13 +170,24 @@ export default class SettingItemMax extends React.PureComponent {
 
         let submit = '';
         if (this.props.submit) {
-            submit = (
-                <SaveButton
-                    saving={this.props.saving}
-                    disabled={this.props.saving}
-                    onClick={this.handleSubmit}
-                />
-            );
+            if (this.props.saveButtonText) {
+                submit = (
+                    <SaveButton
+                        defaultMessage={this.props.saveButtonText}
+                        saving={this.props.saving}
+                        disabled={this.props.saving}
+                        onClick={this.handleSubmit}
+                    />
+                );
+            } else {
+                submit = (
+                    <SaveButton
+                        saving={this.props.saving}
+                        disabled={this.props.saving}
+                        onClick={this.handleSubmit}
+                    />
+                );
+            }
```

Comment:
I think conditional statement here can be omitted. We could just pass the `saveButtonText` props. If it's null then the `SaveButton` component will use its default value.

```
        if (this.props.submit) {
            submit = (
                <SaveButton
                    defaultMessage={this.props.saveButtonText}
                    saving={this.props.saving}
                    disabled={this.props.saving}
                    onClick={this.handleSubmit}
                />
            );
        }
```

Reply:
done. Thanks!